### ISSUE TITLE
[API] Specify if user has voted a track

### ISSUE THREAD

**Issue by pagdot** [skaupper/VirtualJukebox#23]
Specify if a user has voted a track in track queues:

current state:
```
{
    "currently_playing": {
        "track_id": "<track_id>",
        "title": "<track_title>",
        "album": "<album_name>",
        "author": "<author_name>",
        "duration": "<duration>",
        "icon_uri": "<uri>",
        "playing": true|false,
        "playing_for": <playing_for_ms>
    },
    "normal_queue": [
        {
            "track_id": "<track_id>",
            "title": "<track_title>",
            "album": "<album_name>",
            "author": "<author_name>",
            "duration": "<duration>",
            "icon_uri": "<uri>",
            "votes": <nr_of_votes>
        },
        ...
    ],
    "admin_queue": [
        {
            "track_id": "<track_id>",
            "title": "<track_title>",
            "album": "<album_name>",
            "author": "<author_name>",
            "duration": "<duration>",
            "icon_uri": "<uri>"
        },
        ...
    ],
}
```

proposed state:
```
{
    "currently_playing": {
        "track_id": "<track_id>",
        "title": "<track_title>",
        "album": "<album_name>",
        "author": "<author_name>",
        "duration": "<duration>",
        "icon_uri": "<uri>",
        "playing": true|false,
        "playing_for": <playing_for_ms>
    },
    "normal_queue": [
        {
            "track_id": "<track_id>",
            "title": "<track_title>",
            "album": "<album_name>",
            "author": "<author_name>",
            "duration": "<duration>",
            "icon_uri": "<uri>",
            "votes": <nr_of_votes>,
            "has_voted": <has_voted>
        },
        ...
    ],
    "admin_queue": [
        {
            "track_id": "<track_id>",
            "title": "<track_title>",
            "album": "<album_name>",
            "author": "<author_name>",
            "duration": "<duration>",
            "icon_uri": "<uri>"
        },
        ...
    ],
}
```

**Comment by skaupper** [comment]
addressed in #16

**Comment by skaupper** [comment]
I close the issue as soon as the PR is merged
